test/expect/HasDecompTest.test_has_decomposition.expect

```diff
@@ -949,6 +949,8 @@ aten::mps_convolution_backward
 aten::mps_convolution_backward.out
 aten::mps_convolution_transpose_backward
 aten::mps_convolution_transpose_backward.out
+aten::multi_margin_loss_backward
+aten::multi_margin_loss_backward.grad_input
```

Comment:
The following test fails when backward is generated via `py_impl`:

```
pytest -rfxs test/test_decomp.py -k test_has_decomposition -v
```

All other tests should pass.

Reply:
cc @peterbell10

Reply:
Actually, on a second thought, this is expected, so never mind.